### ISSUE TITLE
Plans to maintain?

### ISSUE THREAD

**Issue by Swivelgames** [samicrusader/qomui#13]
This appears to be the "most active fork". Do you have plans to continue to maintain this and overtake the original repo?

**Comment by samicrusader** [comment]
yeah I just haven't had much access to Linux on my desktop for a while as I haven't had much time to get everything reinstalled between the time I graduated school and got a irl job, trying to get all that sorted out at the moment

I want to eventually rewrite at least portions of this since it only works with some (now) odd configurations
